cmd/helm/upgrade.go

```diff
@@ -286,8 +285,6 @@ func (u *upgradeCmd) run() error {
 				force:        u.force,
 				timeout:      u.timeout,
 				wait:         u.wait,
-				description:  "",
```

Comment:
Instead of leaving this unset, should we put in a description that it was rolled back due to the atomic flag?